target/linux/ramips/base-files/etc/board.d/02_network

```diff
@@ -201,6 +201,11 @@ ramips_setup_interfaces()
 	wl-330n)
 		ucidef_set_interface_lan "eth0.1"
 		;;
+	mr200)
+		ucidef_add_switch "switch0" \
+			"0:lan" "1:lan" "2:lan" "3:lan" "6t@eth0"
+		ucidef_set_interface_wan "usb0"
+		;;
```

Comment:
This is not working. the generated network file for wan, ends up being like this:

```
config interface 'wan'                    
        option ifname 'eth0.2'                 
        option proto 'dhcp'               
                                      
config device 'wan_dev'       
        option name 'eth0.2'              
        option macaddr '3c:46:d8:xx:xx:xx'
                                          
config interface 'wan6'                   
        option ifname 'eth0.2'            
        option proto 'dhcpv6'          
```